ProjectLighthouse/Controllers/NewsController.cs

```diff
@@ -1,18 +1,30 @@
+using System.Threading.Tasks;
 using LBPUnion.ProjectLighthouse.Serialization;
 using LBPUnion.ProjectLighthouse.Types.News;
 using Microsoft.AspNetCore.Mvc;
 
-namespace LBPUnion.ProjectLighthouse.Controllers {
+namespace LBPUnion.ProjectLighthouse.Controllers
+{
     [ApiController]
     [Route("LITTLEBIGPLANETPS3_XML/news")]
     [Produces("text/xml")]
-    public class NewsController : ControllerBase {
+    public class NewsController : ControllerBase
+    {
+        [HttpGet("/developer_videos")]
+        public async Task<IActionResult> DeveloperVideos()
```

Comment:
```suggestion
        public IActionResult DeveloperVideos()
```

Reply:
Also, this probably shouldn't be in `NewsController`.

Maybe `DeveloperController`?